ivy_tests/test_ivy/test_functional/test_core/test_manipulation.py

```diff
@@ -288,39 +274,48 @@ def test_reshape(
 
 # roll
 @settings(
-    deadline=750,
+    deadline=1250,
```

Comment:
I think this can be removed, as the default deadline is now 4000 I believe.

Reply:
Removed